### PR TITLE
Create hourly-vs-fixed-bid

### PR DESCRIPTION
**Pull request by BackwardsBadger**
Pull request created

### CODE REVIEW

**Comment by vickynowis** [comment]
looks good to me!

File: hourly-vs-fixed-bid [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    
+Fixed Bid:
+  Pros:
+  - Insures the specific project gets done
+  - Great for small projects or projects with a simple scope
+  Cons:
+  - New adjustments that fall outside of project scope become very costly
+  - It's difficult for a programmer to determine how long certain projects will take
+  - The project cost estimate is ussually rounded significantly up by the programmer to hedge against unforseen issues
+
```
Comment (curfuffled) [inline hourly-vs-fixed-bid:23]: You can also add another con:
- Fixed bids puts a lot of pressure on initial specs, which is problematic if you're still prototyping with users

**Comment by curfuffled** [comment]
Looks great!! 
Had one comment -- see above.
Otherwise, sail on!

File: hourly-vs-fixed-bid (hunk 1 above)
Comment (sdlippel) [inline hourly-vs-fixed-bid:23]: Great job!  Here is link to more discussion and anecdotes on the topic:  http://discuss.joelonsoftware.com/default.asp?biz.5.739195.29

well_done_animation

**Comment by zmschonberger** [comment]
Looks great.  Might also be worth mentioning that all the version control/tracking/project management systems mitigates some of the risk associated with the risk of devs abusing the time & materials contract.

**Comment by ziggawho** [comment]
@curfuffled Nice article - I like how you specify that interests may be misaligned for "hourly", because (as a former consultant for _____), it would be in our interest to max revenues, and open up new problems, rather than fix everything. Having that experience, will definitely do my DD when selecting a programmer to help with my startup!

:+1:

**Comment by BackwardsBadger** [comment]
Added the @curfuffled comment.  Good to go!

**Pull request merged**